src/main/scala/beam/utils/traveltime/FeatureEventHandler.scala

```diff
@@ -57,7 +57,7 @@ class FeatureEventHandler(
 
   def handleEvent(event: Event): Unit = {
     val attrib = event.getAttributes
-    val linkId = Option(attrib.get("link")).map(_.toInt).get
+    val linkId = attrib.get("link").toInt
```

Comment:
This breaks the logic. IIRC `get` can return a null. While at it I'm also wondering if the `toInt` should be wrapped in a Try? But I'm guessing we want an exception if a non-numeric is encountered?